spec/lib/unified_health_data/service_spec.rb

```diff
@@ -935,6 +935,39 @@
         end.to raise_error(StandardError, 'Unknown fetch error')
       end
     end
+
+    context 'LOINC code logging' do
+      before do
+        allow(service).to receive_messages(fetch_access_token: 'token', perform: double(body: notes_sample_response),
+                                           parse_response_body: notes_sample_response)
+        allow(Rails.logger).to receive(:info)
+      end
+
+      it 'logs LOINC code distribution when flipper enabled' do
+        allow(Flipper).to receive(:enabled?).with(:mhv_accelerated_delivery_uhd_loinc_logging_enabled,
+                                                  user).and_return(true)
+
+        service.get_care_summaries_and_notes
+
+        expect(Rails.logger).to have_received(:info).with(
+          {
+            message: 'UHD LOINC code distribution',
+            loinc_code_distribution: '11506-3:3,11488-4:1,4189665:1,18842-5:1,4189666:1,96339-7:1',
+            total_codes: 6,
+            total_records: 6,
```

Comment:
QUESTION: Are they numbers lining up correctly? 

Shouldn't total_records be 8, not 6?